--- asciidoc.py.orig	2009-07-22 23:38:42.000000000 -0700
+++ asciidoc.py	2009-07-22 23:38:55.000000000 -0700
@@ -3326,7 +3326,7 @@
         """ Replace passthough placeholders with the original passthrough
         text."""
         for i,v in enumerate(self.passthroughs):
-            text = text.replace('\x07'+str(i)+'\x07', self.passthroughs[i], 1)
+            text = text.replace('\x07'+str(i)+'\x07', self.passthroughs[i])
         return text
 
 class Macro:
